lib/ansible/plugins/action/copy.py

```diff
@@ -407,12 +407,14 @@ def run(self, tmp=None, task_vars=None):
         local_follow = boolean(self._task.args.get('local_follow', True), strict=False)
 
         result['failed'] = True
-        if ((not source or source is None) and content is None) or dest is None:
-            result['msg'] = "src (or content) and dest are required"
-        elif source is not None and content is not None:
-            result['msg'] = "src and content are mutually exclusive"
+        if (not source or source is None) and content is None:
```

Comment:
Good point. This was what I was originally trying to fix with this PR. I just tested with only `not source` and it passed the tests. Will push shortly.

Please try to merge this today.